docker-compose.yml

```diff
@@ -60,3 +60,12 @@ services:
       timeout: 1m
       retries: 5
       start_period: 10s
+  mock-openid-server:
```

Comment:
Intentional but weird. When striking OIDC endpoints in Unleash, Unleash will make a call to retrieve the well known OIDC config. If it does not find that config, it will fail the call to set OIDC config, but only after it's blocked for a few seconds. It's seriously rude to be striking someone's OIDC config endpoint in tests so this spins up a Python HTTP server which will serve up a valid well known OIDC config for the tests to consume